packages/zfp/meta.yaml

```
@@ -0,0 +1,37 @@
+package:
+  name: zfp
+  version: 1.0.1
+
```

Let's add tags for our CI

```suggestion
  version: 1.0.1
  tag:
    - library
    - static_library
```

---

Addressed in 53286ff13a4316e67a99e6c3a91303b89c34f3ad